temporal-sdk/src/main/java/io/temporal/internal/worker/PollTaskExecutor.java

```diff
@@ -97,6 +97,9 @@ public void process(@Nonnull T task) {
     Preconditions.checkNotNull(task, "task");
     taskExecutor.execute(
         () -> {
+          // TODO: Unclear if we want to keep this metric here, since users might define
```

Comment:
Though we said the metric should be handled in the slot supplier?

Reply:
Yep, this is why the draft said "don't review" :wink: